src/components/settings/SettingsPage.js

```diff
@@ -91,6 +104,16 @@ const SettingsPage = ({ desktop }) => {
           label={<label htmlFor={field}>{label}</label>}
         />
       ))}
+      <LabeledRow
+        label={<label htmlFor="distanceUnit">{t('Units')}</label>}
+        right={
+          <ButtonToggle
+            options={['Metric', 'Imperial']}
+            toggle={updateUnitsSetting}
+            selectedIndex={METRICS_OPTIONS[settings.distanceUnit]}
```

Comment:
It's generally best to name these the standard `onChange`, `value` as in controlled React components.